src/test/java/wolox/training/models/UserTest.java

```diff
@@ -0,0 +1,115 @@
+package wolox.training.models;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import javax.persistence.PersistenceException;
+import org.hibernate.PropertyValueException;
+import org.hibernate.exception.ConstraintViolationException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import wolox.training.repositories.UserRepository;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = Replace.NONE)
+class UserTest {
+
+    @Autowired
+    private TestEntityManager testEntityManager;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @BeforeEach
+    void setUp() {
+    }
+
+    @Test
+    void whenSaveUser_thenReturnUser() {
+        // Given
+        User userTest = new User(null,"Username","David",LocalDate.parse("2004-09-25"), new ArrayList<>());
+        testEntityManager.persist(userTest);
+        testEntityManager.flush();
+
+        //When
+        User userSaved = userRepository.save(userTest);
+
+        //Then
+        assertEquals(userSaved.getUsername(),userTest.getUsername());
+        assertEquals(userSaved.getName(),userTest.getName());
+        System.out.println("whenSaveUser passed");
```

Comment:
This print is not necessary, as it is a test, the result is already shown by the console